lottie-ios/Classes/PublicHeaders/LOTAnimationTransitionController.h

```diff
@@ -41,6 +41,26 @@
                         fromLayerNamed:(NSString *)fromLayer
                           toLayerNamed:(NSString *)toLayer;
 
+/**
+ The initializer to create a new transition animation.
+ 
+ @param animation The name of the Lottie Animation to load for the transition
+ 
+ @param fromLayer The name of the custom layer to mask the fromVC screenshot with.
+ If no layer is specified then the screenshot is added behind the Lottie Animation
+ 
+ @param toLayer The name of the custom layer to mask the toVC screenshot with.
+ If no layer is specified then the screenshot is added behind the Lottie Animation
+ and a fade transition is performed along with the Lottie animation.
+ 
+ @param bundle custom bundle to load animation and images, if no bundle is specified will load
+ from mainBundle
+ */
+- (instancetype)initWithAnimationNamed:(NSString *)animation
+                        fromLayerNamed:(NSString *)fromLayer
+                          toLayerNamed:(NSString *)toLayer
+                              inBundle:(NSBundle *)bundle;
+
```

Comment:
This one already have legacy support. 
https://github.com/airbnb/lottie-ios/pull/238/files/5d52acac8188e2b262c0ab1c38d61e8a8dda9973#diff-b54d62613d6a7f7c40f476c495b7e6f3R40